probe/core/src/main/client/probe.html

```diff
@@ -380,6 +389,44 @@
                         </div>
                     </div>
                     {{/if}}
+                    <div class="form-group">
+                        <label class="col-sm-2 control-label">Scanning information:</label>
```

Comment:
Maybe we should only display this section if there is some content? Or have default message when there is nothing to display (`No scanning information available`)?